database-docker/docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
     volumes:
       - ./db:/db
     environment:
-      - DATABASE_URL=postgres://postgres:postgres@db:5432/dora?sslmode=disable
+      - DATABASE_URL=postgres://postgres:postgres@db:5432/mhq-oss?sslmode=disable
```

Comment:
```suggestion
      - DATABASE_URL=postgres://postgres:postgres@db:5432/middleware?sslmode=disable
```

Reply:
The whole application uses `mhq-oss` in all environments. This shall mean change over their as well

Reply:
ahh pain